nextflow_schema.json

```diff
@@ -47,6 +54,16 @@
                     "description": "Path to CSV file listing the locations of pre-downloaded sourmash databases and lineage files.",
                     "help_text": "Path to CSV file containing the locations of sourmash databases and corresponding lineage files. The CSV should have the column names \"database_path\" and \"lineage_path\" where all the databases are of the same kmer size (by default the workflow runs with kmer size 31)."
                 },
+                "diamond_db": {
+                    "type": "string",
+                    "default": null,
+                    "help_text": "Path to DIAMOND database to compare proteins against. You must make a DIAMOND database prior to using the workflow or use a publicly available one, this workflow does not include the step for making the database."
+                },
+                "diamond_columns": {
+                    "type": "string",
+                    "default": null,
```

Comment:
should this have sensible defaults?

Reply:
if so, i'd also add these defaults to the main config file

Reply:
oh yes can we make the defaults either typical `outfmt 6`, or i think better yet since this is serving our purposes, all of the ones we highlighted previously? Or Or Or all possible would be good too, because why not. i don't think it costs  most to just output all the info

Reply:
here's all of them if you want to go that route: https://github.com/bbuchfink/diamond/blob/7c86d5e6a357ba666f2939e6955a6cc9abb905f3/src/output/blast_tab_format.cpp#L48